c.html.markdown

```diff
@@ -349,26 +355,28 @@ int main (int argc, char** argv)
     break;
   }
   /*
-  using "goto" in C
+    Using "goto" in C
   */
   typedef enum { false, true } bool;
   // for C don't have bool as data type before C99 :(
   bool disaster = false;
   int i, j;
-  for(i=0;i<100;++i)
-  for(j=0;j<100;++j)
+  for(i=0; i<100; ++i)
+  for(j=0; j<100; ++j)
   {
     if((i + j) >= 150)
         disaster = true;
     if(disaster)
-        goto error;
+        goto error;  // exit both for loops
   }
-  error :
+  error: // this is a label that you can "jump" to with "goto error;"
   printf("Error occurred at i = %d & j = %d.\n", i, j);
   /*
-  https://ideone.com/GuPhd6
-  this will print out "Error occurred at i = 51 & j = 99."
+    https://ideone.com/GuPhd6
+    this will print out "Error occurred at i = 51 & j = 99."
   */
+  /* it is generally considered bad practice to do so, except if  */
```

Comment:
Join those two comments and make it one single comment or convert both to single-line `//`-prepended comments?